packages/edit-site/src/components/block-editor/style.scss

```diff
@@ -35,11 +35,16 @@
 
 	.edit-site-visual-editor__editor-canvas {
 		&.is-focused {
-			outline: calc(2 * var(--wp-admin-border-width-focus)) solid var(--wp-admin-theme-color);
-			outline-offset: calc(-2 * var(--wp-admin-border-width-focus));
+			outline-width: var(--wp-admin-border-width-focus);
```

Comment:
I think we should increase this width back to how it was before: `calc(2 * var(--wp-admin-border-width-focus))`. It's hard to notice it at the thinner width.

Reply:
This is x2 on the outline-width and * -3 for the offset. I think this looks good and is easier to see where the focus ring is.

<img width="1083" alt="Site editor with thicker insert focus ring" src="https://github.com/WordPress/gutenberg/assets/967608/9f04a9de-17b3-44b5-8dfe-296825d6369c">

Reply:
> I think we should increase this width back to how it was before: `calc(2 * var(--wp-admin-border-width-focus))`. It's hard to notice it at the thinner width.

But we don't use that thicker focus anywhere else. This is the standard.

Reply:
The size of the thing that's being focused is the difference here. It's way too easy to miss with it wrapping something that takes up almost the entire page. I did a hallway test with my partner and she couldn't even see where the focus went with the thinner ring.

Reply:
I'd echo the instinct to avoid edge-cases on the variables. If we need more indications, potentially additional options are available, such as a stacked outline/halo, a shadow change outside the entire area itself. Unless the regression has been fixed, the scaling of the entire preview is also still missing. 

Another benefit of using the variable is that we can add a toggle or even a stepped slider in the acccessibility settings of preferences to have thicker outlines. The variable would make that relatively trivial to do, since it's already uniformly applied across compontry for that specific reason.

Reply:
I'm fine with whatever approach allows for it to be clear that the canvas is selected. I'm not attached to the approach. The current thinner border isn't enough to communicate the canvas is focused.

Reply:
I agree the inset outline can be tricky to see, especially if the site background is in the same ballpark as the admin theme color. This is a TT4 style variation and it's basically invisible to my eye, even when zoomed in:

<img width="356" alt="Screenshot 2024-05-07 at 13 37 45" src="https://github.com/WordPress/gutenberg/assets/846565/590dcb2f-cae9-4c7c-8b15-aab218dce27c">

Could the outline be outset instead? I don't know if it breaks a11y rules, but this selector works:

`.edit-site-layout__canvas:has(.is-focused) {}`

Reply:
We can't use outset, in its current form at least, as it's hidden via overflow-hidden. Not sure if we can adjust where the focus is applied?

Reply:
![Screenshot 2024-05-08 at 08 56 47](https://github.com/WordPress/gutenberg/assets/1204802/834f2715-3ce9-442f-8207-4841fd4bac18)

This uses a layered focus style to ensure it works while inset.

Reply:
> We can't use outset, in its current form at least, as it's hidden via overflow-hidden

The selector I suggested targets the outer container (only when the button is focused), so is not hampered by `overflow: hidden;`. The idea is to apply the `outline` to that element instead.